kvbc/src/kvbc_app_filter/kvbc_app_filter.cpp

```diff
@@ -309,6 +308,10 @@ uint64_t KvbAppFilter::getValueFromLatestTable(const std::string &key) const {
   return concordUtils::fromBigEndianBuffer<uint64_t>(val->data.data());
 }
 
+TagTableValue KvbAppFilter::getValueFromTagTable(const std::string &tag, uint64_t pvt_eg_id) const {
+  return getValueFromTagTable(tag + kTagTableKeySeparator + concordUtils::toBigEndianStringBuffer(pvt_eg_id));
```

Comment:
Why don't we update the function currently accepting `key` to accept `tag` and `pvt_eg_id`, and concatenate the parameters inside it? i.e., I am trying to understand why do we need to overload the function.

Reply:
You're right, there is actually no use-case for the one-parameter function. I will change it. Thanks for your review!